letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -530,6 +509,10 @@ BootstrapRpmPython3Legacy() {
   fi
 
   BootstrapRpmCommonBase "${python_pkgs}"
+
+  # Try now to enable SCL rh-python36 for systems that are not yet bootstrapped
```

Comment:
While I understand what you mean here, this comment is a bit weird to me because we're calling this function immediately after we've bootstrapped. What about something like:
> Enable SCL rh-python36 after bootstrapping.

Reply:
Comment updated.